packages/manager/modules/iam/src/components/actionSelect/actionSelect.component.js

```diff
@@ -0,0 +1,21 @@
+import controller from './actionSelect.controller';
+import template from './actionSelect.template.html';
+
+import './actionSelect.styles.scss';
+
+export default {
+  bindings: {
+    error: '&?',
+    load: '&?',
+    name: '@?',
+    ngModel: '<',
```

Comment:
This is intentional. We do need the `ng-model` directive on the component and we must require the created controller with `requiredNgModel` to update the model's value. Otherwise form validation would be too complex to wrap